pkg/ccl/importccl/read_import_pgdump.go

```diff
@@ -648,14 +648,18 @@ func (m *pgDumpReader) readFile(
 			if importing && conv == nil {
 				return errors.Errorf("missing schema info for requested table %q", name)
 			}
+			var targetColMapInd []int
```

Comment:
nit, here/below: don't we usually call this `idx` rather than `ind` ?

Reply:
it was called `Ind` in the `INSERT INTO`, so I was form following. I can change both if you want.

Reply:
done!